apps/ledger-live-mobile/src/newArch/features/ModularDrawer/screens/AssetSelection/index.tsx

```diff
@@ -88,17 +88,10 @@ const AssetSelection = ({
   );
 
   useEffect(() => {
-    if (defaultSearchValue === undefined) {
-      return;
-    }
-
     if (availableAssets.length > 0) {
-      const filteredAssets = availableAssets.filter(asset =>
-        asset.name.toLowerCase().includes(defaultSearchValue.toLowerCase()),
-      );
-      setItemsToDisplay(filteredAssets);
+      setItemsToDisplay(availableAssets);
```

Comment:
I don't think we need this anymore now that the search is handled by the backend